Source/inv.cpp

```diff
@@ -1897,15 +1897,45 @@ int8_t CheckInvHLight()
 	return rv;
 }
 
-bool UseScroll(const spell_id spell)
+bool RemoveCurrentSpellScroll(Player &player)

```

Comment:
A bit of an odd name, maybe
```suggestion
bool RemoveActiveScroll(Player &player)
```

Reply:
why not RemoveUsedScroll? :P

Reply:
I tried `ConsumeScroll`, cause that aligns well with `ConsumeSpell` and `ConsumeStaffCharge`.
I hope you like it. 🙂